assets/dev/scss/admin/_dashboard-widget.scss

```diff
@@ -160,3 +160,7 @@
 		}
 	}
 }
+
+.e-dashboard-overview {
```

Comment:
why we still have all the style in the class ID as well ?
why not to replace the `#` to `.` at the original ?